docs/docs/Components/components-vector-stores.md

```diff
@@ -223,6 +223,38 @@ For more information, see the [Couchbase documentation](https://docs.couchbase.c
 |----------------|------------------------|--------------------------------|
 | vector_store   | CouchbaseVectorStore    | A Couchbase vector store instance configured with the specified parameters. |
 
+## CrateDB
+
+This component creates a CrateDB Vector Store with search capabilities.
+For more information, see the documentation about the
+[CrateDB LangChain adapter](https://cratedb.com/docs/guide/integrate/langchain/).
+
+Currently, please install the Langflow CrateDB adapter like this,
+until [issue #157] has been resolved.
+```shell
+uv pip install langflow langchain-cratedb
+```
+[issue #157]: https://github.com/langchain-ai/langchain-postgres/pull/157
```

Comment:
The patch currently does not adjust dependency definitions in `pyproject.toml` by adding the [langchain-cratedb](https://pypi.org/project/langchain-cratedb/) package, because we are waiting for this patch to be included and released.

- https://github.com/langchain-ai/langchain-postgres/pull/157

Reply:
[langchain-postgres v0.0.13](https://github.com/langchain-ai/langchain-postgres/releases/tag/v0.0.13) has been released, so we removed this again.